Dockerfile

```diff
@@ -0,0 +1,166 @@
+ARG unitycatalog_uid=185
+ARG unitycatalog_home="/opt/unitycatalog"
+ARG unitycatalog_repo="${unitycatalog_home}/repo"
+ARG unitycatalog_jar="server/target"
+ARG unitycatalog_etc="etc"
+ARG unitycatalog_bin="bin"
+ARG unitycatalog_user_name="unitycatalog"
+ARG unitycatalog_user_home="home"
+ARG unitycatalog_user_basedir="${unitycatalog_home}/${unitycatalog_user_home}"
+# Specify any custom parameters necessary to generate
+# the Uber-Jar by SBT. 
+# Note: The default allocated heam memory size is too small
+# and will cause the process to fail when attempting to compile
+# and generate the Uber-Jar. Therefore it is important to choose
+# a size large enough for the compiler to run. 
+ARG sbt_args="-J-Xmx5G"
+
+
+
+# ###### STAGE 1 ###### #
+# Building the Uber-Jar #
+#########################
+
+FROM eclipse-temurin:22-jdk-alpine AS build_uc
```

Comment:
Why Java 22, the server only needs Java 17?

Reply:
No, I guess not except the relative freshness of Java 22. It's fine, we can always change if we identify any issues.